Cargo.toml

```diff
@@ -32,7 +32,7 @@ history = []
 dsl = []
 effect = []
 fetch = ["serde", "serde_json", "neq", "future"]
-future = ["wasm-bindgen-futures", "wasm-bindgen", "futures", "web-sys"]
+future = ["wasm-bindgen-futures", "wasm-bindgen", "stdweb", "futures", "web-sys"]
```

Comment:
I think all dependencies except stdweb and futures can be removed because they aren't used anymore.

Reply:
I still see them used in `fetch::request`, maybe we could do two feature flag 
"future-stdweb" and "future-wasm" ?

Reply:
Ah, forgot about that, I'll come up with a solution for that in my own time.

It should be simple enough though, because the fetch feature is intended to only work directly with web_sys.